app/src/hooks/useParaspellApi.tsx

```diff
@@ -152,6 +157,28 @@ const useParaspellApi = () => {
     }
   }
 
+  const validate = async (params: TransferParams): Promise<DryRunResult> => {
+    try {
+      const result = await dryRun(params, params.sourceChain.rpcConnection)
+
+      const dryRunResult: DryRunResult = {
+        type: 'Supported',
+        ...result,
+      }
+
+      return dryRunResult
```

Comment:
We can simplify this kind of stuff

```suggestion
      return {
        type: 'Supported',
        ...result,
      }
```

Reply:
I overlooked this. Thanks